cypress/testComponents/EmailComponentWithFields.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { validateFields } from '../../src/formUtils';
+import { useValidation } from '../../src/useValidation';
+
+const EmailComponentWithFields = (props) => {
+    const email = useValidation(props.email, props.emailValidation);
+
+    useEffect(() => {
+        var validationResult = validateFields([email]);
```

Comment:
`validateFields` should receive object, not array. Right way to write that line should be `constvalidationResult = validateFields({ email });`.

Reply:
Readme needs a fix too.

Reply:
👍

Reply:
@AleksandarDev I will merge the PR when you change this. You can suggest how the documentation can be modified as well (no need to change it now tho).

Reply:
@radovix Fixed and created new issue for documentation